src/remediations/write.integration.js

```diff
@@ -149,7 +149,7 @@ describe('remediations', function () {
         });
 
         test('creates a new remediation with 20k systems', async () => {
-            const name = 'new remediation with issues';
+            const name = `new remediation with issues ${Math.random()}`;
```

Comment:
Can this conflict at some point in future test runs, or, is it cleaned up?

Reply:
The test db is rebuilt at the beginning of the test run, so this was actually fine the way it was.

Reply:
This test was failing and I thought it was because it was creating more than one remediation using the same name in this test but it was actually failing because a previous test in this same file already uses this name: `test('creates a new remediation with issues'`. 
So it thinks the name `new remediation with issues` is a duplicate but if I change it to just `new remediation with issues 2` it passes.